pkgs/applications/misc/plots/default.nix

```diff
@@ -0,0 +1,88 @@
+{ fetchFromGitHub
+, fetchzip
+, gobject-introspection
+, lib
+, libadwaita
+, python3Packages
+, python3
+, wrapGAppsHook4
+, fetchPypi
+}:
+let
+  pyglm = python3Packages.buildPythonPackage rec {
```

Comment:
Why is pyglm defined inside the plots expression instead of separately as its own package?

Reply:
Do you mean that I need to create a separate PR for `pyglm`?

Reply:
No, just a separate commit (and file).